package.json

```diff
@@ -1,7 +1,8 @@
 {
+  "type": "module",
   "dependencies": {
     "@ungap/structured-clone": "^1.2.0",
-    "allotaxonometer": "github:jstonge/allotaxonometer#py-allotax-experimental",
+    "allotaxonometer": "github:jstonge/allotaxonometer#jso-contours",
```

Comment:
This won't call the branch anymore. We'll import the main package, once PR over there is merged.